docs/documentation/overview/customize.html

```diff
@@ -25,7 +25,7 @@
 // Use the existing orange as the danger color
 $danger: $orange
 // Use the new serif family
-$family-primary: $family-serif
+$family-sans-serif: $family-serif
```

Comment:
This doesn't make sense?

Reply:
Sorry, didn't see, problem its, if I declare as you have it will not propagate over the compilation.
If you search for $family-primary inside yours, does not get used.

Sorry for late reply.